common/src/main/java/com/mapbox/vision/common/BaseVisionActivity.kt

```diff
@@ -29,15 +28,16 @@ abstract class BaseVisionActivity : AppCompatActivity() {
         window.addFlags(WindowManager.LayoutParams.FLAG_KEEP_SCREEN_ON)
         super.onCreate(savedInstanceState)
 
-        val board = SystemInfoUtils.getSnpeSupportedBoard()
+        if (!SystemInfoUtils.isVisionSupported()) {
+            Toast.makeText(
```

Comment:
We show _link_ into `Toast` 🤔 may be change it to `Alert dialog`, wdyt?

Reply:
Well, maybe, sounds reasonable

Reply:
@RingerJK I've replaced Toast with activity finish with AlertDialog that is not cancelable to help understand that Vision is not supported on the device